namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-activities-management.tf

```diff
@@ -0,0 +1,119 @@
+resource "kubernetes_secret" "hmpps-activities-management" {
+  metadata {
```

Comment:
Already a secret called this in the namespace - perhaps something like `domain-event-queue`?

Reply:
Ah - it will have -api suffix - but useful to differentiate it?

Reply:
Do you mean because our events may be picked up by non-hmpps subcribers?  e.g. Curious?

Reply:
No, sorry. This is the name of the secret that will appear in the `activities-api-dev` namespace in k8s.  There is already a secret there called `hmpps-activities-management-api` (containing API secrets), so I was suggesting we make this something domain-topic related - that's what it contains - the access/secret/topic ARN for the domain events topic.